CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Mapbox welcomes participation and contributions from everyone.
 # 10.10.0-rc.1
 ## Bug fixes 🐞
 * Trigger repaint after `BitmapWidget` is updated. ([1797](https://github.com/mapbox/mapbox-maps-android/pull/1797))
+* Fix BitmapWidget.setTranslation API to offset relative to its original position. ([1833](https://github.com/mapbox/mapbox-maps-android/pull/1833))
```

Comment:
just a point to consider - when we will be cutting the final v10.10.0 version - that bug fix relative to v10.9.0 changelog might be confusing (as there's no bug if we compare v10.9.0 and v10.0.0). Mayber add a note that this fix is relevant only comparing with beta?

Reply:
yeah, I was thinking to remove this change log entry in stable release note.

Reply:
```suggestion
* Fix a regression from v10.10.0-beta.1 release, the BitmapWidget.setTranslation API should offset relative to its original position. ([1833](https://github.com/mapbox/mapbox-maps-android/pull/1833))
```